openpype/hosts/max/plugins/publish/validate_renderpasses.py

```diff
@@ -0,0 +1,157 @@
+import os
+import pyblish.api
+from pymxs import runtime as rt
+from openpype.pipeline.publish import (
+    RepairAction,
+    ValidateContentsOrder,
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from openpype.hosts.max.api.lib_rendersettings import RenderSettings
+
+
+class ValidateRenderPasses(OptionalPyblishPluginMixin,
+                           pyblish.api.InstancePlugin):
+    """Validates Render Passes before Deadline Submission
+    """
+
+    order = ValidateContentsOrder
+    families = ["maxrender"]
+    hosts = ["max"]
+    label = "Validate Render Passes"
+    optional = True
+    actions = [RepairAction]
+
+    def process(self, instance):
+        if not self.is_active(instance.data):
+            return
+        invalid = self.get_invalid(instance)
+        if invalid:
+            bullet_point_invalid_statement = "\n".join(
+                f"- {err_type}: {filepath}" for err_type, filepath
+                in invalid
+            )
+            report = (
+                "Invalid render passes found.\n\n"
+                f"{bullet_point_invalid_statement}\n\n"
+                "You can use repair action to fix the invalid filepath."
+            )
+            raise PublishValidationError(
+                report, title="Invalid Render Passes")
+
+    @classmethod
+    def get_invalid(cls, instance):
+        """Function to get invalid beauty render outputs and
+        render elements.
+
+        1. Check Render Output Folder matches the name of
+           the current Max Scene, e.g.
+             The name of the current Max scene:
+               John_Doe.max
+             The expected render output directory:
+               {root[work]}/{project[name]}/{hierarchy}/{asset}/
+               work/{task[name]}/render/3dsmax/John_Doe/
+
+        2. Check image extension(s) of the render output(s)
+           matches the image format in OP/AYON setting, e.g.
+               The current image format in settings: png
+               The expected render outputs: John_Doe.png
+
+        3. Check filename of render element ends with the name of
+           render element from the 3dsMax Render Element Manager.
+           e.g. The name of render element: RsCryptomatte
+            The expected filename: {InstanceName}_RsCryptomatte.png
+
+        Args:
+            instance (pyblish.api.Instance): instance
+            filename (str): filename of the Max scene
+
+        Returns:
+            list: list of invalid filename which doesn't match
+                with the project name
+        """
+        invalid = []
+        file = rt.maxFileName
+        filename, ext = os.path.splitext(file)
+        if filename not in rt.rendOutputFilename:
+            cls.log.error(
+                "Render output folder "
+                "doesn't match the max scene name! "
+            )
```

Comment:
Unlike Maya, Max doesn't really have render path file prefixes. 
It is there because the publish job doesn't take the correct directory of the render output, so we need to make sure the render output needs to reset to save in the directory which has the same name with the max scene name. I